web/impact/impact/tests/test_graphql.py

```diff
@@ -0,0 +1,83 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+import json
+from django.urls import reverse
+from impact.tests.factories import ExpertFactory
+from impact.tests.api_test_case import APITestCase
+
+
+class TestGraphQL(APITestCase):
+    url = reverse('graphql')
+
+    def test_query_with_expert(self):
+        with self.login(email=self.basic_user().email):
+            user = ExpertFactory()
+            query = """
+                query {{
+                    expertProfile(id: {id}) {{
+                        user {{
+                            firstName
+                            lastName
+                            email
+                        }}
+                        phone
+                        linkedInUrl
+                        twitterHandle
+                        personalWebsiteUrl
+                        image
+                        title
+                        company
+                        primaryIndustry {{
+                            name
+                        }}
+                        additionalIndustries {{
+                            name
+                        }}
+                        bio
+                        homeProgramFamily {{
+                            name
+                        }}
+                    }}
+                }}
+            """.format(id=user.expertprofile.id)
```

Comment:
this string looks awful.. is there a way to prettify it?

Reply:
In general graphql queries look a bit gnarly at first, they are expressing a lot of information. To my eye, this looks in line with what I would expect from a gql query